### ISSUE TITLE
Bug with automatic formatting for "function-paren-newline" rule

### ISSUE THREAD

**Issue by feross** [eslint/eslint#13801]
**Tell us about your environment**

* **ESLint Version:** 7.12.1
* **Node Version:** 14 LTS
* **npm Version:** 6

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
    "function-paren-newline": ["error", "multiline-arguments"],
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
self._debug(
  'error getting piece %s (offset: %s length: %s) from %s: %s',
  index, chunkOffset, chunkLength, `${wire.remoteAddress}:${wire.remotePort}`,
  err.message
)
```

**What did you expect to happen?**

I expected it to be formatted like this:

```js
self._debug(
  'error getting piece %s (offset: %s length: %s) from %s: %s',
  index,
  chunkOffset,
  chunkLength,
  `${wire.remoteAddress}:${wire.remotePort}`,
  err.message
)
```

**What actually happened? Please include the actual, raw output from ESLint.**

```js
self._debug(
  'error getting piece %s (offset: %s length: %s) from %s: %s',
  index,
  chunkOffset,
  chunkLength,
`${wire.remoteAddress}:${wire.remotePort}`,
err.message
)
```

**Are you willing to submit a pull request to fix this bug?**

Sorry, I don't have time at the moment.

**Comment by ljharb** [comment]
With the indent rule enabled, wouldn't that correct the latter example to your expected example?

**Comment by anikethsaha** [comment]
Thanks for the issue. 


As mentioned by @ljharb , the `indent` rule will fix this. 

[demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IGZ1bmN0aW9uLXBhcmVuLW5ld2xpbmU6IFtcImVycm9yXCIsIFwibXVsdGlsaW5lLWFyZ3VtZW50c1wiXSAqL1xuXG5zZWxmLl9kZWJ1ZyhcbiAgJ2Vycm9yIGdldHRpbmcgcGllY2UgJXMgKG9mZnNldDogJXMgbGVuZ3RoOiAlcykgZnJvbSAlczogJXMnLFxuICBpbmRleCwgY2h1bmtPZmZzZXQsIGNodW5rTGVuZ3RoLCBgJHt3aXJlLnJlbW90ZUFkZHJlc3N9OiR7d2lyZS5yZW1vdGVQb3J0fWAsXG4gIGVyci5tZXNzYWdlXG4pIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6eyJpbmRlbnQiOjIsImZ1bmN0aW9uLXBhcmVuLW5ld2xpbmUiOjJ9LCJlbnYiOnt9fX0=)

**Comment by feross** [comment]
I still reproduce the issue but it only happens when the rest of the rules in `eslint-config-standard@16.0.0` are included. Here's the config to use:

```json
{
  "extends": ["standard"],

  "rules": {
    "function-paren-newline": ["error", "multiline-arguments"]
  }
}
```

I think that perhaps there's a conflict with another rule in `eslint-config-standard`.

**Comment by feross** [comment]
Apologies – this seems to be due to the way we've configured `indent` with `"ignoredNodes": ["TemplateLiteral *"]`.

However, shouldn't the TemplateLiteral still be indented correctly as `TemplateLiteral *` targets only the children?

**Comment by mdjermanovic** [comment]
> However, shouldn't the TemplateLiteral still be indented correctly as `TemplateLiteral *` targets only the children?

Yes, but that also includes [quasis](https://github.com/estree/estree/blob/master/es2015.md#template-literals) (`TemplateElement` nodes ), which are in the given example: `` `${``, `}:${`, and ``}` ``.

Indentation for `` `${`` is ignored since it's a child node of a `TemplateLiteral` node.

**Comment by feross** [comment]
Great, then this is not an ESLint b if. Closing.
